### ISSUE TITLE
Challenges

### ISSUE THREAD

**Issue by richedperson1** [Sayantan-23/Full-Stack-JavaScript-Course#2]
Did you solve challenges which hitesh choudhari give us ? I am just enrol in it

**Comment by Sayantan-23** [comment]
> Did you solve challenges which hitesh choudhari give us ? I am just enrol in it

Which challenge bro?

**Comment by richedperson1** [comment]
Button challenge I solve But I thought it improve it. It's in third video.

**Comment by richedperson1** [comment]
Would like to collaborate with me ? I'm data scientist at harman but wanted to learn web frontend

**Comment by Sayantan-23** [comment]
> Button challenge I solve But I thought it improve it. It's in third video.

No. I'm still unable to understand which third video you are talking about.

**Comment by Sayantan-23** [comment]
> Would like to collaborate with me ? I'm data scientist at harman but wanted to learn web frontend

What type of collaboration bro?

**Comment by richedperson1** [comment]
Button challange link : https://codesandbox.io/s/css-challenges-clxbr1?file=/ButtonChallenges/Challenge_1/index.html:0-474

**Comment by Sayantan-23** [comment]
> Button challange link : https://codesandbox.io/s/css-challenges-clxbr1?file=/ButtonChallenges/Challenge_1/index.html:0-474

No. I only did the assignments and projects of the full stack javascript course in which I'm enrolled.
